### ISSUE TITLE
Determine header level for existing CMS pagers

### ISSUE THREAD

**Issue by ndouglas** [department-of-veterans-affairs/va.gov-cms#16059]
## Description

Followup to #15804.

Tony created a list of pagers in this document: https://docs.google.com/spreadsheets/d/1StwksIw6BG13kNwnob9y39mV4kBeM8I6Fds5f4nQ9Lw/edit#gid=0

However, these cannot be modified in production; they need to be modified as part of a pull request.

Laura suggested that she update this sheet with a column of what the header level should be. After that, Tony can open a single PR or multiple PRs to update the header levels. The number and granularity of followup issue(s) should be left up to Tony's judgment.

## Acceptance Criteria

- [x] Spreadsheet is updated to reflect the correct header levels.
- [x] Followup issue(s) opened to make changes.

**Comment by laflannery** [comment]
Turns out the header levels don't need to change depending on the view the pager is on - everything should be an H2. Follow up ticket [is here](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16077), not sure if it needs to be more detailed though
